apps/dashboard/src/app/(main)/(protected)/projects/[projectId]/users/[userId]/page.tsx

```diff
@@ -0,0 +1,11 @@
+import PageClient from "./page-client";
+
+export const metadata = {
+  title: "User Page",
+};
```

Comment:
can this be the user's display name? You can generate dynamic metadata like this https://nextjs.org/docs/app/building-your-application/optimizing/metadata#dynamic-metadata

Reply:
Can I use hooks in metadata?